src/Console/DevTools.php

```
@@ -85,12 +90,24 @@ public function __construct(
         $this->setDefaultCommand('dev-tools:standards');
         $this->setCommandLoader($commandLoader);
     }
 
+    /**
+     * Resolves the running DevTools version for command metadata.
+     *
+     * @return string the current package version or a safe fallback
+     */
+    private function resolveVersion(): string
+    {
+        return InstalledVersions::getPrettyVersion(self::PACKAGE)
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Composer\InstalledVersions' in method 'resolveVersion'. (undefined)

(StaticAccess)